libafl/Cargo.toml

```diff
@@ -58,6 +58,8 @@ num_enum = { version = "0.5.1", default-features = false }
 typed-builder = "0.9.0" # Implement the builder pattern at compiletime
 ahash = { version = "0.7", default-features=false, features=["compile-time-rng"] } # The hash function already used in hashbrown
 intervaltree = { git = "https://github.com/andreafioraldi/rust-intervaltree", version = "0.2.6", default-features = false, features = ["serde"] }
+digest = "0.9.0"
+backtrace = "0.3.0"
```

Comment:
Needs to be `optional = true` and only enabled for `std`. We can get rid of the `backtrace` in unix (below), then

Reply:
Fixed